src/Emulator/Tabs/ConfigureTab/ConfigureTab.js

```diff
@@ -0,0 +1,136 @@
+import React, { useState, useRef } from 'react';
+import './ConfigureTab.css';
+import DeviceContainer from './DeviceContainer/DeviceContainer';
+
+const MAX_DEVICES = 10;
+
+// Sample responses from currently non-existent API
+// These represent the starting config for a lesson
+const routerConfig = [
+  {
+    name: 'r1',
+    connections: ['s1', 's2'],
+  },
+];
+const switchConfig = [
+  {
+    name: 's1',
+    connections: ['r1', 'h1', 'h2', 'h3'],
+  },
+  {
+    name: 's2',
+    connections: [],
+  },
+];
+const hostConfig = [
+  {
+    name: 'h1',
+    connections: ['s1'],
+  },
+  {
+    name: 'h2',
+    connections: ['s1'],
+  },
+  {
+    name: 'h3',
+    connections: ['s1'],
+  },
+  {
+    name: 'h4',
+    connections: [],
+  },
+];
+
+const getNextNumber = (s) => Number(s.slice(1)) + 1;
```

Comment:
Can you also add a `jsdoc` for this function since slice can be used on an array or a string but in this case it specifically wants a string?

It would look like
```js
/**
 * @param {string} s
 */
const getNextNumber = (s) => Number(s.slice(1)) + 1;
```

Reply:
Added!